docs/source/io.rst

```diff
@@ -17,6 +17,45 @@ Video
 .. autofunction:: write_video
 
 
+Low-level video API
+-------------------
+
+In addition to the :mod:`read_video` function, we provide a high-performance 
+low-level API for more fine-grained control compared to the old API.
```

Comment:
```suggestion
low-level API for more fine-grained control compared to the `read_video` function.
```
I would not call this by "new" / "old" APIs in the documentation, as they won't be "new" anymore in the next release and we will probably forget to update this.